hacker_Rank/May11_Answer.py

```diff
@@ -0,0 +1,21 @@
+from algorithm import Algo
+
+obj = Algo()
+
+
+def solutuion(a):
+    max_value = obj.max_value(a)
+    series = set(item for item in range(1, max_value + 1))
```

Comment:
Can we write simplified way of generating of series ?
``` list(range(1,max_value+1)) ``` 
will not solve problem ?